hazelcast/src/main/resources/hazelcast-full-example.xml

```diff
@@ -1404,6 +1404,17 @@
                     <unique-key-transformation>RAW</unique-key-transformation>
                 </bitmap-index-options>
             </index>
+            <index type="SORTED">
+                <attributes>
+                    <attribute>age</attribute>
+                </attributes>
+                <btree-index>
+                    <memory-tier>
+                        <capacity value="128" unit="BYTES" />
+                    </memory-tier>
+                    <page-size value="1024" unit="BYTES" />
+                </btree-index>
+            </index>
```

Comment:
Not related to this, but can you add a similar sample to hazelcast-fullconfig-without-network.[xml&yaml]